SpecFlowProject10/Form/CalculatorForm.cs

```diff
@@ -134,6 +93,34 @@ public bool CompareResult(string expectedValue)
             return result; // or handle accordingly
         }
 
+
+        public static void CloseInstanceOfApplication(string processName)
```

Comment:
It shouldn't be here

Reply:
I'm not using it anymore, but forgot to remove it from there.
removed the method now